### DISCUSSION TITLE
I made Hindi language common.json file

### DISCUSSION THREAD

**Discussion by hirockstar** [spencerwooo/onedrive-vercel-index#499] [Q&A]
how can i upload that?

**Comment by myl7** [comment]
Great! You can check the docs of next-i18next to get the detailed steps to add a new language.
If you feel it is difficult, you can also just keep your PR and I will handle the remain required stuff.

**Reply by hirockstar** [reply]
thanks bro please handle the remain required stuff. bcz i don't know how to do that.
